src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd

```diff
@@ -0,0 +1,191 @@
+@echo off
+setlocal
+
+set Id=%1
+set IdMaster=Master
+set WcfTestDir=c:\WCFTest
+set RepoHome=c:\git
+set CurrentRepo=%RepoHome%\wcf%Id%
+set MasterRepo=%RepoHome%\wcf%IdMaster%
+set PRServiceName=PRService%Id%
+set WcfServiceName=WcfService%Id%
+set PRServiceMaster=PRService%IdMaster%
+set WcfServiceMaster=WcfService%IdMaster%
+set LogFile="%~dp0%~n0.log"
+set CertExpirationInDays=99
+set ExitCode=0
+set cmd=
+
+if '%1'=='' goto :Usage
+if '%1'=='/?' goto :Usage
+if '%1'=='-?' goto :Usage
+if '%1'=='/help' goto :Usage
+if '%1'=='-help' goto :Usage
+
+:: Make sure IIS appcmd.exe is accessible
+set appcmd=%windir%\System32\inetsrv\appcmd.exe
+if NOT EXIST %appcmd% (
+    echo Could not find: %appcmd%.
+    echo Please make sure IIS is installed.
+    goto :Failure
+)
+
+:: Clean up any existing setup
+echo Deleting WCF repo at %CurrentRepo% if exists and associated application pools and services hosted on IIS...
+if exist %CurrentRepo% rmdir /s /q %CurrentRepo%
+if exist %WcfTestDir% if /I '%MasterRepo%'=='%CurrentRepo%' rmdir /s /q %WcfTestDir%
+%appcmd% delete app "Default Web Site/%PRServiceName%" >nul
+%appcmd% delete apppool %PRServiceName% >nul
+%appcmd% delete app "Default Web Site/%WcfServiceName%" >nul
+%appcmd% delete apppool %WcfServiceName% >nul
+echo Clean up done.
+if /I '%2'=='/c' goto :Done
```

Comment:
>  I tend to like to do command parsing in one go 

I actually thought about this too because that's normally I like to do too. For the case, the code will be changed to something like below. Kind of feel it's a little over kill 😄 

```
if /I '%2'=='/c' set clean=true
... many lines later...
if /I '%clean%'=='true' goto :Done
```

> allows for passing args in any order as the script grows in scope / functionality

This will require more work. I told myself if we need to pass one more argument, I will do it ... and I will move the %2 too 😄.